src/handlers/scan-contributions.ts

```diff
@@ -0,0 +1,117 @@
+import { Context } from "../types";
+
+export async function scanContributions(context: Context) {
+  const { logger, payload, octokit } = context;
+
+  const store: Record<string, Record<string, number>> = {};
+
+  const repo = payload.repository.name;
+  const issueNumber = payload.issue.number;
+  const owner = payload.repository.owner.login;
+  const body = payload.comment.body;
+
+  if (!body.match(/scan-contributions/i)) {
+    return;
+  }
+
+  logger.info("Scanning Events!");
+
+  try {
+    const contributors = await octokit.repos.listContributors({
+      repo,
+      owner,
+    });
+
+    contributors.data.forEach((contributor) => {
+      if (contributor.login) {
+        store[contributor.login] = {};
+      }
+    });
+
+    const issueTimelineEvents = await octokit.paginate(octokit.issues.listEventsForTimeline, { owner, repo, issue_number: issueNumber });
+    const issueEvents = await octokit.paginate(octokit.issues.listEvents, { owner, repo, issue_number: issueNumber });
+    const pullRequestReviewsEvents = await octokit.paginate(octokit.pulls.listReviews, { owner, repo, pull_number: issueNumber });
+    const issueReactionEvents = await octokit.paginate(octokit.reactions.listForIssue, { owner, repo, issue_number: issueNumber });
+    // const issueCommentsReactionEvents = await Promise.all(
```

Comment:
Remove comments.

Reply:
> Remove comments.

@gentlementlegen 
The commented-out code could prove useful to cover other events not captured as pointed out by @0x4007 . I commented them out due to the request limit on the Cloudflare account that runs the plugin. I keep getting a "too many subrequests" error.

Reply:
> Remove comments.

@gentlementlegen 
The commented-out code could prove useful to cover other events not captured as pointed out by @0x4007 . I commented them out due to the request limit on the Cloudflare account that runs the plugin. I keep getting a "too many subrequests" error.